### ISSUE TITLE
[BUG] Error spam after using custom amount.

### ISSUE THREAD

**Issue by ElMoha943** [Pulsih/BankPlus#4]
Steps to reproduce:
1. open bank gui.
2. click on custom amount.
3. type in chat a string instead of a number.

After this every time the player talks it will spam an error in console.

Useful Info:
Plugin version: 2.0
Vault version 1.7.3-b131
TownyChat version 0.88

Error Log:
```
[22:02:05 INFO]: ValenGamerXD issued server command: /bankplus
[22:02:11 ERROR]: Could not pass event AsyncPlayerChatEvent to BankPlus v2.0
java.lang.NumberFormatException: For input string: "HOLA"
        at java.lang.NumberFormatException.forInputString(NumberFormatException.java:67) ~[?:?]
        at java.lang.Long.parseLong(Long.java:714) ~[?:?]
        at java.lang.Long.parseLong(Long.java:839) ~[?:?]
        at me.pulsi_.bankplus.events.PlayerChat.onChat(PlayerChat.java:40) ~[?:?]
        at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor141.execute(Unknown Source) ~[?:?]
        at org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:69) ~[patched_1.16.5.jar:git-Tuinity-"9f3435b"]
        at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:76) ~[patched_1.16.5.jar:git-Tuinity-"9f3435b"]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[patched_1.16.5.jar:git-Tuinity-"9f3435b"]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:624) ~[patched_1.16.5.jar:git-Tuinity-"9f3435b"]
        at io.papermc.paper.adventure.ChatProcessor.post(ChatProcessor.java:225) ~[patched_1.16.5.jar:git-Tuinity-"9f3435b"]
        at io.papermc.paper.adventure.ChatProcessor.processingLegacyFirst(ChatProcessor.java:107) ~[patched_1.16.5.jar:git-Tuinity-"9f3435b"]
        at io.papermc.paper.adventure.ChatProcessor.process(ChatProcessor.java:65) ~[patched_1.16.5.jar:git-Tuinity-"9f3435b"]
        at net.minecraft.server.v1_16_R3.PlayerConnection.chat(PlayerConnection.java:2184) ~[patched_1.16.5.jar:git-Tuinity-"9f3435b"]
        at net.minecraft.server.v1_16_R3.PlayerConnection.c(PlayerConnection.java:2099) ~[patched_1.16.5.jar:git-Tuinity-"9f3435b"]
        at net.minecraft.server.v1_16_R3.PlayerConnection.a(PlayerConnection.java:2037) ~[patched_1.16.5.jar:git-Tuinity-"9f3435b"]
        at net.minecraft.server.v1_16_R3.PacketPlayInChat$1.run(PacketPlayInChat.java:43) ~[patched_1.16.5.jar:git-Tuinity-"9f3435b"]
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
        at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130) ~[?:?]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630) ~[?:?]
        at java.lang.Thread.run(Thread.java:831) [?:?]
[22:02:11 INFO]: [G][Terra][GM]Valen >> HOLA
```

**Comment by Pulsih** [comment]
Ok, i will fix that in the new release, thanks for reporting

**Comment by Pulsih** [comment]
Fixed, i am going to release the v2.1 version rn, you can close this issue, thanks again for reporting!

**Comment by ElMoha943** [comment]
Thanx you!
